genestack_client/scripts/genestack_application_manager.py

```diff
@@ -373,6 +374,10 @@ def run(self):
             print response
 
 
+def match_jar_globs(paths):
+    """ Return a list of files or directories by list of globs """
```

Comment:
The docstring is a phrase ending in a period. https://www.python.org/dev/peps/pep-0257/#id16